.github/labeler.yml

```diff
@@ -1,26 +1,34 @@
 # Add 'kernel' label to any change within Connectors, Extensions, Skills, and tests directories
 kernel:
-  - "dotnet/src/Connectors/**/*"
-  - "dotnet/src/Extensions/**/*"
-  - "dotnet/src/Skills/**/*"
-  - "dotnet/src/IntegrationTests/**/*"
-  - "dotnet/src/SemanticKernel.UnitTests/**/*"
+  - dotnet/src/**/*
+  - python/**/*
+  - java/**/*
 
 # Add 'kernel.core' label to any change within the 'SemanticKernel', 'SemanticKernel.Abstractions', or 'SemanticKernel.MetaPackage' directories
 kernel.core:
   - dotnet/src/SemanticKernel/**/*
   - dotnet/src/SemanticKernel.Abstractions/**/*
   - dotnet/src/SemanticKernel.MetaPackage/**/*
-
-# Add 'python' label to any change within the 'python' directory
-python:
-  - python/**/*
+  - python/semantic_kernel**/*
```

Comment:
I'd keep these separate or introduce python.core labels. I appreciate being able to filter between things that impact skills/connectors/extesions and things that impact the core kernel code.

Reply:
we can use "python.core" but then we should rename "kernel.core" to ".NET.core" or something similar